service/app/domain/Product.scala

```diff
@@ -12,4 +12,5 @@ case class Product(id: Option[Long] = None,
                    lastBoughtAt: Option[DateTime] = None,
                    fillStock: Boolean = false,
                    status: ProductStatus = ProductStatus.ACTIVE,
-                   minQuantity: Long)
+                   minQuantity: Long,
+                   var isInWishlist: Boolean = false)
```

Comment:
Why do you use var here?

Reply:
To be able to assign value to it later, instead it is val.

Reply:
You should use `copy` method.